src/components/FileUpload.tsx

```diff
@@ -55,10 +75,14 @@ const FileUpload = () => {
     setLoading(true);
     const formData = new FormData();
     formData.append('file', pdf);
+    if (fabricLength !== undefined) {
```

Comment:
Is there a way to keep this fabric length stored in the frontend for displaying the iterations?

Reply:
Yep, but what is the purpose of that?

Reply:
the display of the layout for each iteration should look relatively similar if not the same.
the length of the fabric may vary but the width is fixed. Since we're displaying things "sideways" (i.e. the length goes left-right on the screen, the width goes up-down)

Reply:
Ok, when rendering the iterations this variable will still be in scope so we can use it, but I think the issue is in the backend when generating the shape orientation.

[See here](https://github.com/Joshua-Pow/Placement/blob/c375044f95ca5b9f538dfd749c5970069cddaca1/api/index.py#L169C9-L181C10)